src/deluge/gui/views/automation_instrument_clip_view.h

```diff
@@ -105,6 +104,8 @@ class AutomationInstrumentClipView final : public ClipView, public InstrumentCli
 	//not sure how this is used
 	ClipMinder* toClipMinder() { return this; }
 
```

Comment:
refactored code for checking if you're on the automation overview into this function